Makefile

```
@@ -90,6 +91,11 @@ install:
 	ln -srf $(DESTDIR)/$(INSTALL_XROOTD_DIR)/xrootd-stash-origin.cfg $(DESTDIR)/$(INSTALL_XROOTD_DIR)/xrootd-stash-origin-auth.cfg
 	mkdir -p $(DESTDIR)/$(INSTALL_XROOTD_DIR)/config.d
 	install -p -m 0644 $(XROOTD_CONFIGD) $(DESTDIR)/$(INSTALL_XROOTD_DIR)/config.d
+	# XCache Consistency Check
+	mkdir -p $(DESTDIR)/usr/bin
+	mkdir -p $(DESTDIR)/var/lib/xcache-consistency-check
+	install -p -m 0644 src/xcache-consistency-check $(DESTDIR)/usr/bin/xcache-consistency-check
```

```suggestion
	install -p -m 0755 src/xcache-consistency-check $(DESTDIR)/usr/bin/xcache-consistency-check
```
This is executable, right? So it should have execute permissions.